src/core/setToken.ts

```diff
@@ -1,6 +1,6 @@
-import { QueryConstructor } from './spotifyApi';
+import { PersistentApiProperties, Query } from './utils/types';
 
-export const setToken: QueryConstructor<boolean> = (token: string) => (tkn) => {
-  tkn.current = token;
-  return true;
```

Comment:
This removal is causing a test to fail. The test needs to be updated or this line readded.